bin/deploy

```diff
@@ -30,10 +29,16 @@ while [[ $# > 0 ]]; do
         DEPLOY_BRANCH="$2"
         shift
         ;;
+        --verbose)
+        set -x
+        ;;
```

Comment:
Same here (and on line 37 below as well): it looks like some options above have `shift` while others don't. Why and what's the expected behavior?

Reply:
Line 44 has one `shift` for each option already, hence there is only a `shift` instead of a `shift 2` for `--src` as well.